plugins/synapticsmst/synapticsmst-device.c

```diff
@@ -383,39 +383,37 @@ synapticsmst_device_enumerate_device (SynapticsMSTDevice *device,
 				     "Failed to read dpcd from device");
 		goto error_disable_remote;
 	}
-	priv->chip_id = g_strdup_printf ("VMM%02x%02x", byte[0], byte[1]);
+	priv->chip_id = g_strdup_printf ("vmm%02x%02x", byte[0], byte[1]);
```

Comment:
Is this intentional? Will this change the GUIDs?

Reply:
It was intentional, but it shouldn't change the GUIDs - the ones that used it were docks which did `g_ascii_strdown`.